@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/ihadss/fn_ihadssDraw.sqf

```diff
@@ -514,65 +514,39 @@ if (_was == WAS_WEAPON_MSL) then {
     };
     _targpos = _scPos;
 
-    if (isNull _mistargPos) then {
+    if (isNull _missileTarget || (_heli getVariable "fza_ah64_selectedMissile" == "fza_agm114l_wep" && isVehicleRadarOn _heli == false)) then {
```

Comment:
What is this change here achieving?

Reply:
the prevention of the Lock box if lima is selected and the radar is off, that being said i think i added this in error